staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3386,14 +3386,19 @@ type PodSpec struct {
 	HostUsers *bool `json:"hostUsers,omitempty" protobuf:"bytes,37,opt,name=hostUsers"`
 
 	// SchedulingGates is an opaque list of values that if specified will block scheduling the pod.
-	// More info:  https://git.k8s.io/enhancements/keps/sig-scheduling/3521-pod-scheduling-readiness.
+	// If a pod carry non-empty schedulingGates, it will stay in the SchedulingGated state to prevent scheduler
```

Comment:
Rewording the comments per https://github.com/kubernetes/kubernetes/pull/113274#discussion_r1013374289.

@smarterclayton I'd super appreciate it if have cycles to review the new comments.